res/controllers/Hercules-DJControl-MIX-scripts.js

```diff
@@ -0,0 +1,225 @@
+class DJCMixClass {
+    constructor() {
+        ///////////////////////////////////////////////////////////////
+        //                       USER OPTIONS                        //
+        ///////////////////////////////////////////////////////////////
+
+        // How fast scratching is.
+        this.scratchScale = 1.0;
+
+        // How much faster seeking (shift+scratch) is than scratching.
+        this.scratchShiftMultiplier = 4;
+
+        // How fast bending is.
+        this.bendScale = 1.0;
+
+        // DJControl_MIX_scripts.js
+        //
+        // ****************************************************************************
+        // * Mixxx mapping script file for the Hercules DJControl MIX.
+        // * Author: DJ Phatso and Kerrick Staley
+        // * Version 1 (Jan 2023)
+        // * Based on Hercules DJControl Starlight mapping released with Mixxx v2.3.0
+        // *  -Remapped LOOP and SAMPLER section according to DJControl MIX layout
+        // *  -Added Master Volume and Headphone Volume
+        // *  -Removed superfluous LED configuration (not present on DJControl MIX)
+        // * Forum: https://mixxx.discourse.group/t/hercules-contrl-mix-mapping/26581/
+        // * Wiki: https://mixxx.org/wiki/doku.php/
+
+
+        this.kScratchActionNone = 0;
+        this.kScratchActionScratch = 1;
+        this.kScratchActionSeek = 2;
+        this.kScratchActionBend = 3;
+    }
+
+    init() {
+        this.scratchButtonState = true;
+        this.scratchAction = {
+            1: this.kScratchActionNone,
+            2: this.kScratchActionNone
+        };
+
+        // Vinyl button LED On.
+        midi.sendShortMsg(0x91, 0x03, 0x7F);
+
+        // Set effects Levels - Dry/Wet
+        engine.setParameter("[EffectRack1_EffectUnit1_Effect1]", "meta", 0.6);
+        engine.setParameter("[EffectRack1_EffectUnit1_Effect2]", "meta", 0.6);
+        engine.setParameter("[EffectRack1_EffectUnit1_Effect3]", "meta", 0.6);
+        engine.setParameter("[EffectRack1_EffectUnit2_Effect1]", "meta", 0.6);
+        engine.setParameter("[EffectRack1_EffectUnit2_Effect2]", "meta", 0.6);
+        engine.setParameter("[EffectRack1_EffectUnit2_Effect3]", "meta", 0.6);
+        engine.setParameter("[EffectRack1_EffectUnit1]", "mix", 1);
+        engine.setParameter("[EffectRack1_EffectUnit2]", "mix", 1);
+
+        // Ask the controller to send all current knob/slider values over MIDI, which will update
+        // the corresponding GUI controls in MIXXX.
+        midi.sendShortMsg(0xB0, 0x7F, 0x7F);
+    };
```

Comment:
```suggestion
    }
```
Please delete this semicolon